### PR TITLE
Convert file URLs to proper file paths

### LINKED ISSUE #180: Wrong path to nodeDebug.js
Hi,

trying to run a Node.js application in eclipse I get the following error (note the double `C:\`):
```
module.js:549
    throw err;
    ^

Error: Cannot find module 'C:\C:\eclipse\plugins\org.eclipse.wildwebdeveloper_0.3.0.201906060852\language-servers\node_modules\node-debug2\out\src\nodeDebug.js'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Function.Module.runMain (module.js:693:10)
    at startup (bootstrap_node.js:188:16)
    at bootstrap_node.js:609:3
```
Looking at the corresponding launch configuration there is the entry `listEntry`:
```
<launchConfiguration ...>
...
    <listAttribute key="org.eclipse.lsp4e.debug.model.ATTR_DSP_ARGS">
      <listEntry value="/C:/eclipse/plugins/org.eclipse.wildwebdeveloper_0.3.0.201906060852/language-servers/node_modules/node-debug2/out/src/nodeDebug.js"/>
...
    </listAttribute>
</launchConfiguration>
```
My guess is the the leading slash causing the double `C:\`. 

In the classes files NodeAttachDebugDelegate (in line 56) and NodeRunDAPDebugDelegate (in line 86) I've found the following line:
```
wc.setAttribute(DSPPlugin.ATTR_DSP_ARGS, Collections.singletonList(FileLocator.toFileURL(getClass().getResource("/language-servers/node_modules/node-debug2/out/src/nodeDebug.js")).getPath())); //$NON-NLS-1$
```
Maybe the used `org.eclipse.core.runtime.FileLocator` or org.eclipse.core.internal.runtime.Activator.getURLConverter() causing this behavior.

Is there a possible workaround by setting an environment variable?

Installation details:
- Windows 10
- Eclipse IDE Version 2019-06 (4.12.0)
- Wild Web Developer - Web edition in Eclipse IDE (Experimental)	0.4.1.201906280859	org.eclipse.wildwebdeveloper.feature.feature.group	Eclipse Wild Web Developer project

Kind regards

Sven

### PR DESCRIPTION
**Pull request by PyvesB**
Fixes #180.

Signed-off-by: Pierre-Yves B. PyvesDev@gmail.com

### CODE REVIEW

File: org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
  * Contributors:
  *   Mickael Istria (Red Hat Inc.) - initial implementation
  *   Gautier de Saint Martin Lacaze - Issue #55 Warn missing or incompatible node.js
+ *   Pierre-Yves B. - Issue #196 NullPointerException when validating Node.js version
```
Comment (PyvesB) [inline org.eclipse.wildwebdeveloper/src/org/eclipse/wildwebdeveloper/InitializeLaunchConfigurations.java]: Not directly related, but I forgot to update this in #197.

**Comment by mickaelistria** [comment]
Thanks a lot for this good patch.
But the ECA check requires that the mail address in signed-off-by footer of commit message gets between <..>. Can you please update that, amend and push --force on your topic branch to update this commit?
Sorry about the noise!

**Comment by PyvesB** [comment]
Done! Usually Eclipse gets that right when using EGit, however, I was struggling due to this [bug](https://bugs.eclipse.org/bugs/show_bug.cgi?id=548716) and had to add the `Signed-off-by` manually via command line - and omitted the `<..>`. 😄

**Comment by mickaelistria** [comment]
Build pass locally. I'm investigating (with moderated success so far) how to make the tests more reliable.
Thanks!

**Pull request merged**